### PR TITLE
symbol table European Voynich Alphabet closes#1471

### PR DESCRIPTION
**Pull request by t-m-z**

### CODE REVIEW

File: lib/application/main_menu/licenses.dart [hunk 1]
```diff
@@ -135,6 +135,7 @@ class _LicensesState extends State<Licenses> {
           [i18n(context, 'symboltables_sanluca_title'), 'Leadermassimo (wikimafia.it) (CC BY-SA 4.0)'],
           [i18n(context, 'symboltables_solmisation_title'), 'www.breitkopf.de (Personal Use)'],
           [i18n(context, 'symboltables_sprykski_title'), '(Personal Use)'],
+          [i18n(context, 'symboltables_voynich_title'), 'VonHaarberg, (WikiCommons, CC BY-SA 4.0)'],
```
Comment (S-Man42) [inline lib/application/main_menu/licenses.dart:138]: please also add in licence file

Comment (t-m-z) [reply lib/application/main_menu/licenses.dart:138]: ? I probably missed something. Do we have a special licence file addionally?

Comment (t-m-z) [reply lib/application/main_menu/licenses.dart:138]: done

**Comment by S-Man42** [comment]
conflicts

**Pull request merged**